tests/strict_dbl_quotes.test/t02.req.out

```diff
@@ -0,0 +1,21 @@
+('------- PHASE 1 -------'='------- PHASE 1 -------')
+("1"='1')
+("x"='x')
+(x='1')
+("y"='y')
+('------- PHASE 2 -------'='------- PHASE 2 -------')
+[SELECT "1"] failed with rc -3 double-quoted string literal: "1"
+[SELECT "x"] failed with rc -3 double-quoted string literal: "x"
+(x='1')
+[SELECT "y" FROM t1 WHERE x = 1] failed with rc -3 double-quoted string literal: "y"
```

Comment:
This should have passed. We're only forbidding double-quoted string literals. Double-quoted identifiers are perfectly valid (and also SQL standard).

Reply:
Which query should have passed here?  The one on line 10?  For that one, the t1 table has no column "y".  Several others are similar.

Reply:
Just added some more tests to verify that valid double-quoted identifiers are working correctly with the tunable enabled.

Reply:
I stand corrected. I thought "y" was a column in t1.